sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EditEventPage.java

```diff
@@ -49,4 +49,6 @@
   public static final By DISEASE_COMBOBOX = By.cssSelector("#disease div");
   public static final By SOURCE_TYPE_COMBOBOX = By.cssSelector("#srcType div");
   public static final By TYPE_OF_PLACE_COMBOBOX = By.cssSelector(" #typeOfPlace div");
+  public static final By NEW_TASK_BUTTON = By.cssSelector("#taskNewTask");
+  public static final By EDIT_TASK_O = By.cssSelector("#edit-task-0");
```

Comment:
this needs to be changed because we can have multiple tasks attached to an event, and taking based on index like this is not ok 🗡️ 

1. Take as a reference this from framework: 
_String CONTACT_RESULTS_UUID_LOCATOR = "[title = '%s']";_
2. Change the variable with string, and try to navigate to the edit button based on task type or something which can be used unique
3. Change cssSelector with Id

Reply:
@BojteTamas  I had a call with Richard, looks like we won't be able to identify an unique task, but the last added task is always on top on index 0. Are you ok to keep Richard's approach or do you have other ideas?

Reply:
@rdutu-vg totally agree, with this approach, only a small suggestion to put `first` instead `0`, to actually see the index.

Reply:
And please add the `BUTTON`